src/common/config/defaultPreferences.ts

```diff
@@ -43,6 +43,7 @@ const defaultPreferences: ConfigV3 = {
     lastActiveTeam: 0,
     downloadLocation: getDefaultDownloadLocation(),
     startInFullscreen: false,
+    downloads: [],
```

Comment:
Just curious, are these downloads that will be resumed once the application closes?

If not, I feel like we can just either store them in a variable somewhere, or if it needs to be a file, then maybe a different one.

The `config.json` file is mostly for persistent user settings.

Reply:
Well I'm wondering if they need to be stored at all. If a download is finished or cancelled, I don't think we need to store that information after the app has closed. If it's in progress, then I could see it, but that assumes that we're able to resume the download afterwards.